dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -203,19 +204,22 @@ public void testComponent_staticLinkageCheckResult() throws IOException, Parsing
     try (InputStream source = Files.newInputStream(grpcAltsHtml)) {
       Document document = builder.build(source);
 
-      Nodes staticLinkageCheckMessage = document.query("//pre[@class='jar-linkage-report']");
+      Nodes staticLinkageCheckMessage = document.query("//p[@class='jar-linkage-report']");
       Assert.assertEquals(1, staticLinkageCheckMessage.size());
```

Comment:
I know `Nodes` doesn't implement `Iterable<Node>`, but could we (in a separate PR) have some static method that creates an `ImmutableList<Node>` from a `Nodes`, so that we could `assertThat(nodes).comparingElementsUsing(NODE_VALUES).containsExacltly(something)` or do other useful things?